brave/brave-core
# Auto-Contribute list only contains included publishers

Fixes: https://github.com/brave/brave-browser/issues/1175

**This requires** https://github.com/brave-intl/bat-native-ledger/pull/117 **for testing**

## Submitter Checklist:


## Test Plan:
Essentially this can be best described as a regression test for the exclusion/restoration of excluded publishers.

1. Build Brave from this branch with storage
2. Enable Payments
3. Add 6 - 10 sites to the auto-contribute table
4. Exclude 1 or 2 sites
5. Ensure that the number of excluded sites updates appropriately
6. Restore the excluded publishers
7. Ensure that the restoration message goes away
8. Re-excluded the same publishers
9. Ensure that accurate count is reflected.

## Reviewer Checklist:
